perfmetrics/scripts/continuous_test/gcp_ubuntu/periodic_experiments/experiments_configuration.json

```diff
@@ -22,7 +22,13 @@
       "config_name": "MaxConnsPerHost40",
       "gcsfuse_flags": "--implicit-dirs --max-conns-per-host 40 --debug_fuse --debug_gcs",
       "branch": "experiments_dashboard",
+      "end_date": "2023-05-30 05:30:00+00:00"
```

Comment:
Also, I see an issue with "end_date" here i.e. this end_date will not be reflected in the bigquery table and hence in the dashboard.

Reply:
Can you elaborate on this a little? Why will it not be reflected?

Reply:
When build.sh will run, it will only pick the experiments whose end_date >= today's date, so experiment MaxConnsPerHost40 will not be picked and hence it's end_date will continue to be the one when it was enabled last. Because of which the dashboard will also show that last updated date.

Reply:
As discussed, we note down internally and will pick this task of adding logic to update end_date of all experiments in config file every time an experiment is run later.